### PR TITLE
Added unittest to check for specific repo files

### LINKED ISSUE #52: Add unittest to check for specific repo files
This repo should have the following list of files included:
- ./Docker or docker/Docker
- ./docker-compose.yml or ./docker/docker-compose.yml
- ./.env_sample
- ./.gitignore
- ./.travis.yml
- ./.codeclimate.yml
- ./CHANGELOG.md
- ./CODE_OF_CONDUCT.md
- ./CONTRIBUTING.md
- ./.github/ISSUE_TEMPLATE
- ./LICENSE.md
- ./.github/PULL_REQUEST_TEMPLATE
- ./README.md
- ./TROUBLESHOOTING.md
- ./USAGE.md
- ./USE_CASES.md

This PR is only asking for tests of the existence of these files, if the files do not exist when you run the tests - do not worry about the tests not passing. We will identify this and create a new PR for the issue.
Thank you!

### PR DESCRIPTION
**Pull request by mptap**
# Fixes #52

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Added unittest to check for specific repo files as mentioned in https://github.com/sendgrid/nodemailer-sendgrid-transport/issues/52

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

@mbernier edit: fixed link at the top

### CODE REVIEW

**Comment by mbernier** [comment]
I merged in recent changes, which should allow the build to continue.

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+var assert      = require('assert');
+var should      = require('should');
```
Comment (mbernier) [inline test/test.js]: @mptap it looks like line 2 is throwing an error:
```
> mocha
(node:4228) [DEP0006] DeprecationWarning: child_process: options.customFds option is deprecated. Use options.stdio instead.
module.js:538
    throw err;
    ^
Error: Cannot find module 'should'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/test/test.js:2:19)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at /home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/lib/mocha.js:184:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/lib/mocha.js:181:14)
    at Mocha.run (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/lib/mocha.js:393:31)
    at Object.<anonymous> (/home/travis/build/sendgrid/nodemailer-sendgrid-transport/node_modules/mocha/bin/_mocha:380:16)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
npm ERR! Test failed.  See above for more details.
```

are we using this module in this test? Might be ok to just remove the import?

**Comment by mptap** [comment]
@mbernier: Updated, thanks. It's working now.

**Comment by thinkingserious** [comment]
Hello @mptap,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**